std_msgs/Float64 rho_xy_12
std_msgs/Float64 rho_xy_13
std_msgs/Float64 rho_xy_23